config/environments/test.rb

```diff
@@ -7,6 +7,8 @@
   # and recreated between test runs. Don"t rely on the data there!
   config.cache_classes = true
 
+  config.assets.js_compressor = :uglifier
```

Comment:
A side-effect of this change is that developers running tests locally will probably have a slower test suite. Before, the JS in test mode was only  concatenated. Now, it is also uglified (which is not fast). The suite locally for me was about 50 seconds.

The work-around here would be to create a new environment just for building these assets, but I fear it would get out of sync over time. This seems like a good iterative step.